### PR TITLE
Update menufront.sma

### PR DESCRIPTION
**Pull request by 9iky6**
Just a minor update the plugin and change work in lang-key: you can now add the file through the menu items with the lang-key.

### CODE REVIEW

File: plugins/menufront.sma [hunk 1]
```diff
@@ -142,15 +142,15 @@ stock bool:get_clcmd_flags(const search_command[], &flags)
...
 }
+
 public actionMenu(id, key)
 {
 	switch (key)
 	{
 		case 8: displayMenu(id, ++g_menuPosition[id])
 		case 9: displayMenu(id, --g_menuPosition[id])
-		default: client_cmd(id, "%s", g_menuCmd[g_menuPosition[id] * 8 + key])
+		default: client_cmd(id, "%s", g_menuCmd[g_menuPosition[id] * 8 + key]) // may need to replace : amxclient_cmd?
```
Comment (xPaw) [inline plugins/menufront.sma]: If there are no plugins that rely on actually executing client commands, then `amxclient_cmd` should work just fine.

**Comment by xPaw** [comment]
:+1:

**Comment by gullubhai** [comment]
:+1: Good :D

**Comment by Arkshine** [comment]
Looks ok. Thanks.

**Pull request merged**